tests/wasmbus.rs

```diff
@@ -1250,6 +1302,7 @@ expected: {expected_labels:?}"#
                 module_http_port,
                 module_nats_client.clone(),
                 &mut module_redis_client,
+                true,
```

Comment:
like mentioned above, I'd make the function *return* the values and assert after the call directly here